data/virt_autotest/virt_logs_collector.sh

```diff
@@ -144,10 +149,10 @@ EOF
 	fi
 }
 
-#This function supports collecting supportconfig from both host and guest. The argument target_type will be given 'host' or 'guest'
+#This function supports collecting supportconfig or sosreport from both host and guest. The argument target_type will be given 'host' or 'guest'
 #Will resort to guest virsh console if collecting from guest ssh failed. Collecting logs from host only supports local host
-#Typical usage: collect_supportconfig logs_folder host or collect_supportconfig logs_folder guest guest_ip guest_domain_name
-function collect_supportconfig() {
+#Typical usage: collect_support_report logs_folder host or collect_support_report logs_folder guest guest_ip guest_domain_name
+function collect_support_report() {
```

Comment:
Is there other existing calls to the orginal function? Can you figure out aother function name which indicates what it does better -- it is not easy for people to map support_report with the real logs?

Reply:
Changed it to collect_system_log_and_diagnosis